aspnetcore/signalr/javascript-client/samples/3.x/SignalRChat/wwwroot/chat.js

```diff
@@ -37,12 +37,8 @@ document.addEventListener("DOMContentLoaded", () => {
         }
     };
     
-    connection.onclose(function (error) {
-        if (error) {
-            console.log(`Connection closed with error: ${error}`);
-        } else {
-            console.log("Connection closed.");
-        }
+    connection.onclose(async () => {
+        await start();
```

Comment:
😦 this was just changed earlier and now you reverted it https://github.com/dotnet/AspNetCore.Docs/pull/22694